src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -54,11 +55,24 @@ public TypedParameter(string name, ProtoCore.Type type, AssociativeNode defaultV
         public ProtoCore.Type Type { get; private set; }
 
         /// <summary>
-        /// Returns default value of the parameter.
+        /// Returns default value of the parameter as AST node.
         /// </summary>
+        [JsonIgnore]
         public AssociativeNode DefaultValue { get; private set; }
 
         /// <summary>
+        /// Returns fully qualified string representation of AST node.
+        /// </summary>
+        [JsonProperty("DefaultValue")]
+        public string DefaultValueString
+        {
+            get
+            {
+                return defaultValueString;
+            }
```

Comment:
@ke-yu if there are no default values this will return null.   Is this what CoGS expects or possibly an empty string?